app/model.py

```diff
@@ -0,0 +1,30 @@
+from mongoengine import *
+
+
+connect('linguadb')
+
+
+class Languages(Document):
+    language = StringField(required=True)
+    code = StringField(required=True, max_length=3)
```

Comment:
I would not enforce the max length here to be 3

Reply:
Code here refers to ISO 639-2 which is a three letters word.